### DISCUSSION TITLE
no mileage report for some cars?

### DISCUSSION THREAD

**Discussion by nuznormal** [flobz/psa_car_controller#329] [Q&A]
is it possible that my 2022 peugeot e208 will not report mileage or is it a controller problem?
I can see other info (range, battery level, charging status and control preconditioning) but no mileage.
“mileage” : null

**Comment by guinola** [comment]
I have the same problem with an e-208.
Any solution?

**Reply by nuznormal** [reply]
It started to work after a few days. The problem was only at the beginning when first installed it. Either the account needed to activate in some way or it was a psa issue.

**Reply by guinola** [reply]
In the app the service e-remote is activated, but the Telemaintance is in the process of being activated, which is probably why I don't get all the data. Thanks.

**Comment by duanra08** [comment]
In HAOS, I have the same problem since the 1st october
